sdk/typescript/src/rpc/client.ts

```diff
@@ -36,16 +36,18 @@ export class JsonRpcClient {
     const req = { method, args };
 
     const response = await this.request(method, args);
-    const [err] = validate(response, struct);
-
-    if (err) {
-      console.warn(
-        new RPCValidationError({
-          req,
-          result: response,
-          cause: err,
-        }),
-      );
+
+    if (process.env.SUI_TEST === 'true') {
+      const [err] = validate(response, struct);
+      if (err) {
+        console.warn(
```

Comment:
While we're making this change can we just make this throw, and remove the console warn mock that we use the make validation errors throw?